src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -290,16 +290,19 @@ def getReplicaInfoForBlocks(self, **kwargs):
 
         return result
 
-    def getPFN(self, site, lfns, protocol=None):
+    def getPFN(self, site, lfns, protocol=None, protocol_domain='ALL', operation=None):
```

Comment:
Stefano, can you please adopt the lower camel case variable name? So changing `protocol_domain` to `protocolDomain`?
I've been trying to avoid this never ending mix of variable naming conventions in WMCore.

Reply:
You are right. There's no need to keep same name as RUCIO API in our code. Changed.